THIRD-PARTY-NOTICES

```diff
@@ -62,7 +62,7 @@ SOFTWARE.
 
 The following NPM package may be included in this product:
 
- - @yext/search-core@2.5.0
+ - @yext/search-core@2.5.4
```

Comment:
I'm pretty sure we want to merge this into a hotfix branch first and not directly into main / master right?

Reply:
Huh, I assumed it would automatically create the hotfix branch since I have that as upstream on my local. Updated